ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Config.scala

```diff
@@ -26,6 +26,7 @@ final case class Config(
     listTestSuites: Boolean,
     shuffleParticipants: Boolean,
     partyAllocation: PartyAllocationConfiguration,
+    ledgerClockGranularityMs: Int
```

Comment:
Trailing commas made your diff nicer. Please try to pay it forward. 🙂 

```suggestion
    ledgerClockGranularityMs: Int,
```

Reply:
addressed